### ISSUE TITLE
fillna() issue 

### ISSUE THREAD

**Issue by khannadheeraj** [abhishekkrthakur/approachingalmost#71]
At page 89 while doing label encoding. Nan value is filled with "NONE" which is string so it is giving error "argument must be.of string or float"   . I can be fix either by 
`df.ord_2.fillna(np.nan)`
or
`lbl_enc.fit_transform(df.ord_2.values.astype(str))`
